elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -88,7 +85,7 @@ public void commit() {
             T object = entityClass.newInstance();
             deferredTasks.add(() -> session.persist(object));
             return object;
-        } catch (InstantiationException | IllegalAccessException e) {
+        } catch (java.lang.InstantiationException | IllegalAccessException e) {
```

Comment:
There was a direct collision with imports due to optimization of imports for hibernate. Therefore I had to explicitly namespace the class path here. Same with hibernate5